jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java

```diff
@@ -86,6 +83,7 @@ private static String driverPath(String driverName) {
     public static String  LATEST_VERSION = "LATEST";
     public static String DRIVER_VERSION = LATEST_VERSION;
     public static String PRELATEST_VERSION = "PRELATEST";
+    public static String ARGUMENTS_PROPERTY = "arguments";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4683015)